tests/test_dataloaders.py

```diff
@@ -0,0 +1,87 @@
+from datasets import Dataset, load_dataset
```

Comment:
Have integrated into the evaluator tests! 

re: test split selection — this implicitly already happens in the second test I just added because there is no split explicitly chosen. Let me know if you had something else in mind though, and I can add that as well!